### ISSUE TITLE
Lostfilm Indexer categories are broken

### ISSUE THREAD

**Issue by stasivanov** [Jackett/Jackett#9854]
### Environment

**OS**: LibreElec 9.2.4

**Jackett Version**: 0.16.1740.0

**Last Working Jackett Version**: v0.16.1709

**Are you using a proxy or VPN?** no

### Description

So, it seems that categories list changed for Lostfilm. Probably smth is wrong with them. I have an issue when adding indexer to Sonarr. Please check https://github.com/Sonarr/Sonarr/issues/4018 as well. 

### Logged Error Messages

So Sonarr returns an error 
```
20-10-15 12:56:36.6|Debug|Api|[GET] /api/v3/config/indexer: 200.OK (3 ms)
20-10-15 12:56:36.6|Debug|Api|[GET] /api/v3/indexer: 200.OK (43 ms)
20-10-15 12:56:37.5|Debug|Api|[GET] /api/v3/indexer/schema: 200.OK (27 ms)
20-10-15 12:56:44.4|Debug|Api|[POST] /api/v3/indexer/action/newznabCategories: 200.OK (13 ms)
20-10-15 12:56:56.1|Debug|Torznab|Downloading Feed http://192.168.1.70:9117/api/v2.0/indexers/lostfilm/results/torznab/api?t=tvsearch&cat=5030,5040&extended=1&apikey=(removed)&offset=0&limit=100
20-10-15 12:56:56.2|Warn|Torznab|Unable to connect to indexer

[v3.0.4.982] NzbDrone.Core.Indexers.Torznab.TorznabException: Torznab error detected: lostfilm does not support the requested query. Please check the capabilities (t=caps) and make sure the search mode and categories are supported.
  at NzbDrone.Core.Indexers.Torznab.TorznabRssParser.PreProcess (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x000b8] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\Torznab\TorznabRssParser.cs:42 
  at NzbDrone.Core.Indexers.RssParser.ParseResponse (NzbDrone.Core.Indexers.IndexerResponse indexerResponse) [0x0000d] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\RssParser.cs:55 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].FetchPage (NzbDrone.Core.Indexers.IndexerRequest request, NzbDrone.Core.Indexers.IParseIndexerResponse parser) [0x00052] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:308 
  at NzbDrone.Core.Indexers.HttpIndexerBase`1[TSettings].TestConnection () [0x00044] in M:\BuildAgent\work\63739567f01dbcc2\src\NzbDrone.Core\Indexers\HttpIndexerBase.cs:342 

20-10-15 12:56:56.2|Warn|SonarrErrorPipeline|Invalid request Validation failed: 
 -- : Unable to connect to indexer, check the log for more details
20-10-15 12:56:56.2|Debug|Api|[POST] /api/v3/indexer: 400.BadRequest (59 ms)
20-10-15 12:56:57.6|Debug|Api|[GET] /api/v3/health: 200.OK (2 ms)
20-10-15 12:56:57.6|Debug|Api|[GET] /api/v3/system/status: 200.OK (4 ms)
20-10-15 12:56:57.7|Debug|Api|[GET] /api/v3/diskspace: 200.OK (104 ms)
20-10-15 12:56:58.9|Debug|Api|[GET] /api/v3/log/file: 200.OK (4 ms)
```

Here are categories for the Lostfim

```
<?xml version="1.0" encoding="UTF-8"?>
<caps>
  <server title="Jackett" />
  <searching>
    <search available="yes" supportedParams="q" />
    <tv-search available="yes" supportedParams="q,season,ep" />
    <movie-search available="yes" supportedParams="q" />
    <music-search available="no" supportedParams="" />
    <audio-search available="no" supportedParams="" />
    <book-search available="no" supportedParams="q" />
  </searching>
  <categories>
    <category id="2000" name="Movies">
      <subcat id="2010" name="Movies/Foreign" />
      <subcat id="2020" name="Movies/Other" />
      <subcat id="2030" name="Movies/SD" />
      <subcat id="2040" name="Movies/HD" />
      <subcat id="2045" name="Movies/UHD" />
      <subcat id="2050" name="Movies/3D" />
      <subcat id="2060" name="Movies/BluRay" />
      <subcat id="2070" name="Movies/DVD" />
      <subcat id="2080" name="Movies/WEBDL" />
    </category>
  </categories>
</caps>
```

So, maybe an issue is in Naming "Movies". Here is what Sonarr support responded. 
```
Ask on the Jackett support channel whether it should be TV, or both. They'd be able to change it on their end.
Be sure to mention your Jackett version to them.
```

**Comment by stasivanov** [comment]
I would also want to test on the latest version, 1740, but an update fails to me with an error
```
Error performing update.
System.Threading.Tasks.TaskCanceledException: The operation was canceled.
 ---> System.IO.IOException: Unable to read data from the transport connection: Operation canceled.
 ---> System.Net.Sockets.SocketException (125): Operation canceled
   --- End of inner exception stack trace ---
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.GetResult(Int16 token)
   at System.Net.Security.SslStream.<FillBufferAsync>g__InternalFillBufferAsync|215_0[TReadAdapter](TReadAdapter adap, ValueTask`1 task, Int32 min, Int32 initial)
   at System.Net.Security.SslStream.ReadAsyncInternal[TReadAdapter](TReadAdapter adapter, Memory`1 buffer)
   at System.Net.Http.HttpConnection.FillAsync()
   at System.Net.Http.HttpConnection.CopyToContentLengthAsync(Stream destination, UInt64 length, Int32 bufferSize, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnection.ContentLengthReadStream.CompleteCopyToAsync(Task copyTask, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.HttpConnection.ContentLengthReadStream.CompleteCopyToAsync(Task copyTask, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionResponseContent.SerializeToStreamAsync(Stream stream, TransportContext context, CancellationToken cancellationToken)
   at System.Net.Http.HttpContent.LoadIntoBufferAsyncCore(Task serializeToStreamTask, MemoryStream tempBuffer)
   at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
   at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 165
   at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /home/vsts/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 199
   at Jackett.Common.Services.UpdateService.DownloadRelease(List`1 assets, Boolean isWindows, String version) in /home/vsts/work/1/s/src/Jackett.Common/Services/UpdateService.cs:line 248
   at Jackett.Common.Services.UpdateService.CheckForUpdates() in /home/vsts/work/1/s/src/Jackett.Common/Services/UpdateService.cs:line 139
```
But that's not about this topic...

**Comment by SuberSeb** [comment]
I have this issue too.

**Comment by stasivanov** [comment]
Just a guess it's broken since https://github.com/Jackett/Jackett/commit/dcf339a6eb9ce54fd57900d59eda9e1bb7877710

**Comment by garfield69** [comment]
fixed dcf339a6eb9ce54fd57900d59eda9e1bb7877710 which should have defined the category as TV not Movies.
the next release of Jackett due out in about 12 hours or so will include the fix.

**Comment by garfield69** [comment]
Jackett 0.16.1757
